src/Microsoft.Extensions.Configuration.AzureAppConfiguration/OfflineFileCache.cs

```diff
@@ -192,9 +180,9 @@ public void Export(AzureAppConfigurationOptions options, string data)
                             using (var writer = new Utf8JsonWriter(memoryStream))
                             {
                                 writer.WriteStartObject();
-                                writer.WriteString(dataProp, Convert.ToBase64String(encryptedBytes));
-                                writer.WriteString(hashProp, dataHash);
-                                writer.WriteString(scopeProp, scopeHash);
+                                writer.WriteString(dataProp, encryptedData);
+                                writer.WriteString(scopeProp, _scopeToken);
+                                writer.WriteString(expiryProp, expiryString);

```

Comment:
>expiryString [](http://example.com/codeflow?start=63&length=12)

The variable naming is unconventional, usually type isn't a part of the name and should be avoided. But I see it's because there are two "expiry" here. Could be avoided by not having expiryString at all and just doing the tostring here.

Reply:
How about expiryToken? 
The same naming problem exists in `Import` too, and there we can't avoid an intermediate string while reading from file.

Reply:
I saw it in Import, but didn't comment because I didn't see a way to avoid it. Here though, it looked like only one variable was needed.

Reply:
Okay, removed the `expiryString` variable from Export.